index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio skeleton</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css?family=Poppins&display=swap" rel="stylesheet" />
+</head>
+
+<body>
+  <main>
+
+    <div class="container"></div>
+      <header>
+        <nav class="navbar">
+          <div class="header-top-art">
+            <img src="./images/image_geometry_1.png" alt="top left art"/>
+          </div>
+       
+          <div class="menu-ham">
+            <img src="/images/icon-menu.svg" alt="ham icon menu"/>
```

Comment:
- [ ] The hamburger button's image is not displayed. Kindly check the path to the image
![andym-logo-screenshot](https://user-images.githubusercontent.com/44057863/164608443-15dfd1aa-7a7e-468d-b83d-7076c57f87fc.png)
.

Reply:
solved a dot  before folder url was not present